libs/ui-lib/lib/common/components/ui/formik/CodeField.tsx

```diff
@@ -40,7 +64,7 @@ const CodeField = ({
     setMonacoSubscription(
       monacoEditor.onDidChangeModelContent(() => {
         setTouched(true);
-        setValue(monacoEditor.getModel()?.getValue(), true);
+        isValidValue ? setValue(monacoEditor.getModel()?.getValue(), true) : setValue('', true);
```

Comment:
The content flashes for a little bit before we remove it for being invalid. Is there are way to prevent that?